src/app/views/dashboard/games/manage/game/packages/list/list.ts

```diff
@@ -57,6 +59,7 @@ export default class RouteDashGamesManageGamePackagesList extends BaseRouteCompo
 		const payload = await Api.sendRequest('/web/dash/developer/games/packages/' + route.params.id);
 
 		if (payload.packages && !payload.packages.length) {
+			// TODO(collaborators) a page for when there are no packages but no permission to create any
```

Comment:
what do you think we should do? I don't know if we should have a whole new page for this, or just have the list page show "No packages" and don't do the redirect here if they don't have perms

Reply:
did this, didn't test. can you give it a check?